app/src/test/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/ConfirmationUseCaseTest.kt

```diff
@@ -16,11 +17,9 @@ import io.reactivex.schedulers.TestScheduler
 import org.junit.Test
 import java.util.concurrent.TimeUnit
 
-class VoiceAnalysisUseCaseTest {
+class ConfirmationUseCaseTest {
 
-    private val dataRepository = mock<DataRepository> {
-        on { updateVoiceAnalysisOption(any()) }.doReturn(Completable.complete())
-    }
+    private val dataRepository = mock<DataRepository>()
     private val randomDigitsList = listOf(1, 2, 3, 4)
     private val confirmationId = randomDigitsList.joinToString("")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c7380d74fc32b4ddceef02c8d9a561008c986f4f/app/src/test/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/ConfirmationUseCaseTest.kt#L24"></span>Private property rxWebSocketClient is unused.</td>
    </tr>
  </tbody>
</table>